crates/swift-integration-tests/src/async_function.rs

```diff
@@ -13,7 +13,8 @@ mod ffi {
     extern "Rust" {
         async fn rust_async_return_null();
         async fn rust_async_reflect_u8(arg: u8) -> u8;
-        async fn rust_async_reflect_string(string: String) -> String;
+        // TODO: this is broken because RustString is not Sendable.
+        // async fn rust_async_reflect_string(string: String) -> String;
```

Comment:
The `TODO` describes the problem but not what we should do about it.

Do we need to do research? Do we already know the solution? Is the problem unsolvable? etc.
Let's make it more clear here what the next step is on this TODO.